### DISCUSSION TITLE
[feature request] auxiliary FAN

### DISCUSSION THREAD

**Discussion by PatrickDenis** [OrcaSlicer/OrcaSlicer#597] [General]
Great Work ! really like what you did !!!! was able to make Vzbot with 330 and 235 !! working like a charm.

Question, i see in the slicer there is the auxiliary FAN option, i would love to know how to use it with the RSCS secondary fan on the VZbot ?

Any Idea of where or how to implement it with klipper ? Bambu use the same secondary fan on the side.

Thanks in advance

**Comment by stonemandave** [comment]
Have you found an answer to this question.
